### PR TITLE
Added ZONE variable to the info string which informs what provider is…

### PR DESCRIPTION
**Pull request by bestrand**
… being used

### CODE REVIEW

**Comment by roberthbailey** [comment]
Not all providers have a definition of ZONE. I think this will cause kube-up to break if you run it with vagrant as your provider (for instance).

**Comment by golvteppe** [comment]
Thanks for pointing out @roberthbailey, I will fix it and come back.

File: cluster/kube-up.sh [hunk 1]
```diff
@@ -34,7 +34,12 @@ fi
 source "${KUBE_ROOT}/cluster/kube-env.sh"
 source "${KUBE_ROOT}/cluster/kube-util.sh"
 
-echo "... Starting cluster using provider: $KUBERNETES_PROVIDER" >&2
+
+if [ -z "${ZONE-}" ]; then
+  echo "... Starting cluster using provider: $KUBERNETES_PROVIDER" >&2
```
Comment (roberthbailey) [inline cluster/kube-up.sh]: please use curlys when expanding variables (e.g. `${KUBERNETES_PROVIDER}`).

**Comment by roberthbailey** [comment]
@k8s-bot ok to test

**Comment by roberthbailey** [comment]
LGTM.

**Pull request merged**